src/OrchardCore.Modules/OrchardCore.GitHub/Recipes/GithubAuthenticationSettingsStep.cs

```diff
@@ -27,7 +27,7 @@ public async Task ExecuteAsync(RecipeExecutionContext context)
                 return;
             }
             var model = context.Step.ToObject<GitHubLoginSettingsStepModel>();
-            var settings = await _githubAuthenticationService.LoadSettingsAsync();
+            var settings = await _githubAuthenticationService.GetSettingsAsync();
```

Comment:
`GetSettingsAsync()` returns an immutable settings object that won't be possible to update in the next lines, so this should stay Load*. Same for all the *Step classes.